docs/source/model_design/vivarium_features/automated_v_and_v/index.rst

```diff
@@ -0,0 +1,245 @@
+  And then add it to the list of decorators above.
+
+.. _automated_v_and_v:
+
+.. role:: underline
+    :class: underline
+
+=========================================================
+Automated verification and validation (V&V)
+=========================================================
+
+.. contents::
+   :local:
+
+.. note::
+
+  This is a proposal for how to automate parts of the verification and validation process.
+  It builds on some work that has already been done on the probabilistic record linkage (PRL) project but is not
+  yet routine practice on other simulations: integration testing.
+
+  It also covers some extensions to integration testing which would allow it to automate more V&V checks.
+  As of April 2023, these are in the process of being implemented on the PRL project.
+
+Background/Motivation
+---------------------
+
+:ref:`The current process for verification and validation (V&V) <vivarium_v_and_v_process>`
+works well but can be quite labor intensive.
+Additionally, due to the long runtime of simulations and the number of people involved,
+each V&V cycle generally takes *at least* one day, which makes it difficult to find and fix issues quickly.
+
+Automating more of the V&V process would help us catch issues faster and with less work.
+
+**We do not need to automate all possible V&V checks.** The more potential issues we can catch quickly and
+automatically, the better.
+This will also free up researcher V&V time to do more open-ended exploration of
+results, which is difficult or impossible to automate.
+
+Automated V&V
+-------------
+
+Automating V&V requires transforming the manual process into something that can be executed by a computer
+and generate an unambiguous verdict on whether or not there is an issue.
+
+.. note:: 
+  In the future, we could explore having an in-between result, where the automated process flags
+  a *potential* issue for human review.
+  For now, we stick to a binary result.
+
+There are some process questions to work out about this transformation:
+
+* Who investigates the cause of automated V&V failures -- research or engineering? Does it depend on the type of failure?
+* How frequently does automated V&V need to be run, and with what population size/number of draws/etc?
+    * Should we consider separate "tiers" of automated checks that require different amounts of computational resources?
+* What sort of sensitivity/specificity tradeoff do we want for our automated V&V?
+  This likely depends on both of the previous questions.
+
+PRL case study: integration tests
++++++++++++++++++++++++++++++++++
+
+We have automated some V&V checks on the PRL project by adding integration tests
+to `the simulation repository <https://github.com/ihmeuw/vivarium_census_prl_synth_pop/>`_.
+These tests run the simulation for a given number of time steps and check certain conditions
+with explicit Python :code:`assert` statements, which fail in unexpected situations.
```

Comment:
```suggestion
with explicit Python :code:`assert` statements, which fail when the model exhibits unexpected behavior.
```
Changing this because before it could be interpreted that we cannot predict when our tests will fail :)